Gemfile.lock

```diff
@@ -170,16 +170,6 @@ GEM
       coderay (~> 1.0)
       method_source (~> 0.8)
       slop (~> 3.4)
-    pry (0.9.12.6-i386-mingw32)
```

Comment:
Can you help me understand why you removed these?

Reply:
this was the automatic result of a fresh bundle install.

Reply:
okay. Let me pull down this branch first and check that it won't break a windows build :)

Reply:
👍 if it does I can figure out how to get this into the Gemfile so anyone who starts fresh will end up with the correct dependencies. Keep me posted!